src/common/options/mds.yaml.in

```diff
@@ -1450,3 +1450,12 @@ options:
   - mds
   flags:
   - runtime
+- name: mds_disable_metrics_update_rank0
+  type: bool
+  level: advanced
+  desc: Disable updating metrics to rank0 periodically.
+  default: false
+  services:
+  - mds
+  flags:
+  - runtime
```

Comment:
`mds_disable_metrics_update_rank0` with a default of `false` is a double negation ... and a bit hard to make sense of.
Could you change the option name to something like `mds_send_metrics_to_rank0` and then decide on the default value.
Sending metrics amounts to sending an "update" anyway. Hence, I've dropped the "update" word from the suggestion.

Reply:
Makes sense, will fix it.  There still has some problems with this PR I will fix it if this patch still needed.
Thanks.